lambda_src/batch_locking_backends/dynamodb.py

```diff
@@ -0,0 +1,94 @@
+import os
+from typing import Dict, Text, List, Any
+import boto3
+from json import dumps
+from hashlib import md5
+
+from botocore.exceptions import ClientError
+from ..utils import LOG, ResponseType
+
+AWS_REGION = os.environ.get(
+    "AWS_REGION", "us-west-2"
+)  # Placeholder while in dev TODO: change as variable/header
```

Comment:
not sure I understand this "TODO" -- what will it change to in prod?

Reply:
I'd replicated it from [here](https://github.com/Snowflake-Labs/geff/blob/09721ec1c09c009f9bab6d3da061fa5fe87f664f/lambda_src/drivers/destination_s3.py#L16), with the assumption that this instance will need the change too.